llvm/lib/Target/RISCV/RISCVInsertVSETVLI.cpp

```diff
@@ -1255,6 +1258,12 @@ void RISCVInsertVSETVLI::emitVSETVLIs(MachineBasicBlock &MBB) {
         MachineOperand &VLOp = MI.getOperand(getVLOpNum(MI));
         if (VLOp.isReg()) {
           // Erase the AVL operand from the instruction.
```

Comment:
This comment applies to the `setReg` and `setIsKill` call. Insert your code above it.